src/main/core-api/java/com/mysql/cj/conf/PropertyDefinitions.java

```diff
@@ -662,6 +662,9 @@
                 new BooleanPropertyDefinition(PropertyKey.acceptAwsProtocolOnly, DEFAULT_VALUE_FALSE, RUNTIME_NOT_MODIFIABLE,
                         Messages.getString("ConnectionProperties.acceptAwsProtocolOnly"), "0.1.0", CATEGORY_HA, Integer.MAX_VALUE),
 
+                new BooleanPropertyDefinition(PropertyKey.allowXmlUnsafeExternalEntity, DEFAULT_VALUE_FALSE, RUNTIME_NOT_MODIFIABLE,
+                        Messages.getString("ConnectionProperties.allowXmlUnsafeExternalEntity"), "0.2.0", CATEGORY_HA, Integer.MAX_VALUE),
```

Comment:
I don't think it's CATEGORY_HA. It should be something generic.

Reply:
Would CATEGORY_JDBC be better or CATEGORY_SECURITY or something else?

Reply:
CATEGORY_SECURITY seems a good option.